ups/deploy_testing_distrib.cfg

```diff
@@ -0,0 +1,13 @@
+# -*- python -*-
+
+import lsst.sconsUtils
+
+dependencies = dict(
+)
+
+config = lsst.sconsUtils.Configuration(
+    __file__,	
+    libs=[],
+    hasDoxygenInclude=False,
+    hasSwigFiles=False,
+    )
```

Comment:
@timj I opened a backlog story to fixup packages which don't need a `.cfg`.  https://jira.lsstcorp.org/browse/DM-6393